client/src/Components/Accordion/Accordion.js

```diff
@@ -0,0 +1,68 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { IconContext } from 'react-icons';
+import { BsChevronRight, BsChevronDown } from 'react-icons/bs';
+import ProductItem from '../Products/ProductItem';
+
+const AccordionSection = styled.div`
+  margin: 15px 0px;
+`;
+
+const Wrap = styled.div`
+  background: #ce0e55;
+  color: #FFFFFF;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  cursor: pointer;
+  h1{
+    letter-spacing: 0.5px;
+    font-size:18px;
+    padding-left:2rem;
+  }
+  span {
+    margin-right: 1.5rem;
+  }
+`;
+
+const Accordion = ({ categories, products }) => {
+    const [clicked, setClicked] = useState(false);
```

Comment:
if you are going to store a number in the state, provide default number value not boolean

Reply:
yes, pushed the changes